### DISCUSSION TITLE
How to compile clang?

### DISCUSSION THREAD

**Discussion by RohitKushvaha01** [termux/termux-packages#22139] [Q&A] [answered]
I tried compiling termux clang using the [build-package.sh](https://github.com/termux/termux-packages/blob/master/build-package.sh) script from Termux-packages in Docker, but it's failing. Am I missing something?
![image](https://github.com/user-attachments/assets/b602adbe-f3c7-4786-9e4c-e371c9ad665c)

**Comment by twaik** [comment, marked as answer]
Clang is a subpackage of libllvm, it is not being built separately.
